site/src/pages/docs/server-annotated-service.mdx

```diff
@@ -324,6 +324,23 @@ public class MyAnnotatedService {
 }
 ```
 
+If you specify query delimiter while building an annotated service, or a `Delimiter` annotation with a query
```

Comment:
```suggestion
If you specify query delimiter while building an annotated service, or a <type://@Delimiter> annotation with a query
```

Reply:
Just a question, what is difference between ``Delimiter`` and `<type://@Delimiter>`?

Reply:
Good question. `<type://@Delimiter>` will link to the Javadoc like:
<img width="147" alt="image" src="https://user-images.githubusercontent.com/1866157/163784860-4e06af8e-f55c-4c7e-8104-de560ad68852.png">